README.md

```diff
@@ -1367,6 +1367,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Nishant-Jangid](https://github.com/1010nishant)
   - [Nishant Shrivastva](https://github.com/shrinishant)
   - [Nishant Siddh](https://github.com/nishant-siddh)
+  - [Nisha Singhal](https://github.com/nishaYO)
```

Comment:
Please add your name in line 1366, after Nirvana.
Let us know if you need any help with that. 🙂

Reply:
Added a the name as suggested in new branch☺

Reply:
@nishaYO I have merged your recent PR. you can close with PR:)